src/deals/list/deals.scss

```diff
@@ -340,3 +413,32 @@ pbutton button.secondary {
     }
   }
 }
+
+@media screen and (max-width: 400px) {
+  .page.deals {
+    .section {
+      &.all-deals {
+        .grid.dealGrid {
+          .row {
+            grid-template-areas:
+              "daos daos daos"
+              "type type type"
+              "size size status";
+            grid-template-columns: 2fr 1fr 0.5fr;
+            .title, .age {
+              display: none;
```

Comment:
The reason for not showing title is because of the space, but imo, is Title not one of the most important piece of information for a Deal?

Though looking at the reality, the titles just reflect the DAO names.
![image](https://user-images.githubusercontent.com/30693990/174786365-c34c992c-5910-4fbc-b5e3-d7571c7f9d36.png)

If you agree on including titles in the mobile view for tables, could you create a separate ticket?
Regardless, I'm fine with skipping it now.